scripts/zones/La_Theine_Plateau/npcs/Vicorpasse.lua

```diff
@@ -27,7 +27,7 @@ function onTrigger(player,npc)
 	if(player:getCurrentMission(SANDORIA) == THE_RESCUE_DRILL) then
 		local MissionStatus = player:getVar("MissionStatus");
 		
-		if(MissionStatus == 4) then
+		if(MissionStatus >= 0 and MissionStatus <= 4) then

```

Comment:
bg wiki: `First talk to Galaihaurat at (E-6)`

ffxiclopedia: `Head to La Theine Plateau and speak to Galaihaurat next to a low wall at the bottom-middle box of (E-6) to start the rescue drill exercise.`

zam: `The guard sends you to La Theine Plateau to receive some rescue training. To get there go to the sw corner of West Ronfaure and zone into La Theine Plateau. Go to E-6 and speak with the NPC (Galaihaurat)`

IGN: `After receiving the mission from a gate guard, head to West Ronfaure and zone into La Theine Plateau at (F-12). Speak to Galaihaurat at (E-6) `

GamerEscape: `Galaihaurat at (E-6). Speak to him to start the rescue drill`